.github/CODEOWNERS

```
@@ -0,0 +1,128 @@
+/packages/cisco_duo @elastic/security-external-integrations
+/packages/cisco_ftd @elastic/security-external-integrations
+/packages/cisco_ios @elastic/security-external-integrations
+/packages/cisco @elastic/security-external-integrations
+/packages/cisco_meraki @elastic/security-external-integrations
+/packages/cisco_nexus @elastic/security-external-integrations
+/packages/cisco_secure_endpoint @elastic/security-external-integrations
+/packages/cisco_umbrella @elastic/security-external-integrations
+/packages/cloudflare @elastic/security-external-integrations
+/packages/cockroachdb @elastic/integrations
+/packages/containerd @elastic/integrations
+/packages/crowdstrike @elastic/security-external-integrations
+/packages/cyberark @elastic/security-external-integrations
+/packages/cyberarkpas @elastic/security-external-integrations
+/packages/cylance @elastic/security-external-integrations
+/packages/docker @elastic/integrations
+/packages/elastic_agent @elastic/elastic-agent-control-plane
+/packages/elasticsearch @elastic/integrations
+/packages/etcd @elastic/integrations
+/packages/f5 @elastic/security-external-integrations
+/packages/fireeye @elastic/security-external-integrations
+/packages/fleet_server @elastic/ingest-management
+/packages/fortinet @elastic/security-external-integrations
+/packages/gcp @elastic/security-external-integrations
+/packages/github @elastic/security-external-integrations
+/packages/google_workspace @elastic/security-external-integrations
+/packages/haproxy @elastic/integrations
+/packages/hashicorp_vault @elastic/security-external-integrations
+/packages/hid_bravura_monitor @elastic/security-external-integrations
+/packages/httpjson @elastic/security-external-integrations
+/packages/iis @elastic/integrations
+/packages/imperva @elastic/security-external-integrations
+/packages/infoblox @elastic/security-external-integrations
+/packages/iptables @elastic/security-external-integrations
+/packages/journald @elastic/integrations
+/packages/juniper_junos @elastic/security-external-integrations
+/packages/juniper @elastic/security-external-integrations
+/packages/juniper_netscreen @elastic/security-external-integrations
+/packages/juniper_srx @elastic/security-external-integrations
+/packages/kafka @elastic/integrations
+/packages/keycloak @elastic/security-external-integrations
+/packages/kibana @elastic/integrations
+/packages/kubernetes @elastic/integrations
+/packages/linux @elastic/integrations
+/packages/log @elastic/integrations
+/packages/logstash @elastic/integrations
+/packages/mattermost @elastic/security-external-integrations
+/packages/microsoft_defender_endpoint @elastic/security-external-integrations
+/packages/microsoft_dhcp @elastic/security-external-integrations
+/packages/microsoft @elastic/security-external-integrations
+/packages/microsoft_sqlserver @elastic/security-external-integrations
+/packages/mimecast @elastic/external-security-integrations
```

This has the name of the team wrong.
```suggestion
/packages/mimecast @elastic/security-external-integrations
```

---

FWIW I noticed it when looking at this output which groups the packages by owner.

`yq eval-all '. as $item ireduce({}; .[$item.owner.github] += [$item.name])' packages/*/manifest.yml`

```yaml
elastic/security-external-integrations:
  - 1password
  - akamai
  ...
elastic/integrations:
  - activemq
  ...
```

---

And here's my `yq` to generate the codeowners: 😄 

`yq eval-all '. as $item ireduce([]; . += "/packages/" + $item.name + " @" + $item.owner.github ) | .[]' packages/*/manifest.yml`